### PR TITLE
Switch from NSLog to FIRFADLogger

### PR DESCRIPTION
**Pull request by jeremydurham**

### CODE REVIEW

File: FirebaseAppDistribution/Sources/FIRAppDistribution.m [hunk 1]
```diff
@@ -194,8 +193,8 @@ - (void)fetchReleases:(FIRAppDistributionUpdateCheckCompletion)completion {
                                                  NSString *_Nonnull idToken,
                                                  NSError *_Nullable error) {
     if (error) {
-      NSLog(@"Error getting fresh auth tokens. Will sign out tester. Error: %@",
-            [error localizedDescription]);
+      FIRFADInfoLog(@"Error getting fresh auth tokens. Will sign out tester. Error: %@",
```
Comment (schnecle) [inline FirebaseAppDistribution/Sources/FIRAppDistribution.m:196]: Should we make this a FIRFADErrorLog?

File: FirebaseAppDistribution/Sources/FIRAppDistribution.m [hunk 2]
```diff
@@ -266,7 +266,7 @@ - (void)handleOauthDiscoveryCompletion:(OIDServiceConfiguration *_Nullable)confi
        appDistributionSignInCompletion:(void (^)(NSError *_Nullable error))completion {
   if (!configuration) {
     // TODO: Handle when we cannot get configuration
-    NSLog(@"ERROR - Cannot discover oauth config");
+    FIRFADInfoLog(@"ERROR - Cannot discover oauth config");
```
Comment (schnecle) [inline FirebaseAppDistribution/Sources/FIRAppDistribution.m:269]: Same question here

Comment (schnecle) [reply FirebaseAppDistribution/Sources/FIRAppDistribution.m:269]: We shouldn't need the ERROR prefix then.

File: FirebaseAppDistribution/Sources/FIRAppDistribution.m [hunk 3]
```diff
@@ -296,13 +296,13 @@ - (void)handleOauthDiscoveryCompletion:(OIDServiceConfiguration *_Nullable)confi
...
             // User cancelled auth flow
-            NSLog(@"Tester cancelled sign in flow");
+            FIRFADInfoLog(@"Tester cancelled sign in flow");
             signInError =
                 [self NSErrorForErrorCodeAndMessage:FIRAppDistributionErrorAuthenticationCancelled
                                             message:kAuthCancelledErrorMessage];
           } else {
             // Error in the auth flow
-            NSLog(@"Tester sign in error - %@", [error localizedDescription]);
+            FIRFADInfoLog(@"Tester sign in error - %@", [error localizedDescription]);
```
Comment (schnecle) [inline FirebaseAppDistribution/Sources/FIRAppDistribution.m:305]: log level here too - this is likely my own mistakes coming back to get me.

File: FirebaseAppDistribution/Sources/FIRAppDistribution.m (hunk 2 above)
Comment (jeremydurham) [reply FirebaseAppDistribution/Sources/FIRAppDistribution.m:269]: Yes, will switch these to error! :)

File: FirebaseAppDistribution/Sources/FIRAppDistribution.m (hunk 1 above)
Comment (jeremydurham) [reply FirebaseAppDistribution/Sources/FIRAppDistribution.m]: Done!

File: FirebaseAppDistribution/Sources/FIRAppDistribution.m (hunk 3 above)
Comment (jeremydurham) [reply FirebaseAppDistribution/Sources/FIRAppDistribution.m]: Done!

**Pull request merged**